Tests/HumanizeTests.cs

```diff
@@ -74,17 +118,17 @@ public void Test2ndDigitRangePositive(int number, string expected)
         [DataRow(-1999001, "nearly minus 2 million")]
         public void Test2ndDigitRangeNegative(int number, string expected)
         {
-            Assert.AreEqual(expected, Translations.Humanize(number));
+            Assert.AreEqual(expected, Translations.Humanize(number, false));
         }
 
         [DataTestMethod]
-        [DataRow(111000, "111000")]
+        [DataRow(111000, "111,000")]
         [DataRow(111100, "just over 111 thousand")]
         [DataRow(111200, "over 111 thousand")]
         [DataRow(111700, "nearly 112 thousand")]
         public void TestNumbersWith3DigitMantissa(int number, string expected)
         {
-            Assert.AreEqual(expected, Translations.Humanize(number));
+            Assert.AreEqual(expected, Translations.Humanize(number, false));
         }
     }
```

Comment:
I would say no. My plan all along was to have zero Cottle impact, and enhance Humanise() for slavic and other number systems entirely in C#-land.

Reply:
I'd say that whether or not for force integer mantissas is a localisation decision that hinges around whether *the language in question* supports constructions like "2.4 million". Thus, it sits in the language translation domain, which we represent with REGX files, rather than the Cottle domain.

Reply:
To put it another way, (I'm thinking aloud here), Cottle users shouldn't need to think about this stuff. They should be able to call `{Humanise(n)}` and get a gramattically correct phrase for their language -- end of story! :)

Reply:
Fair, though in that case I'd be tempted to ask the user to enter a formula or orderMultiplier rather than simply recording a true / false preference in the .resx.

As an example... I consider `two thousand four hundred` to be more proper than `2.4 thousand` even though I also consider `two million four hundred thousand` and `2.4 million` to be equally proper. So I might want the value to be "true" for the `1E3` orderMultiplier value but "false" for higher orderMultiplier values.

Reply:
I'm afraid I'm going to push back here, sorry. I'd rather not make this user-visible because for one thing it would cause difficulties for Russian and any similar languages we may encounter going forward, where integer mantissas are a necessity for correct grammar. Besides, it's an awful lot of added complexity to expose.